lib/board.rb

```diff
@@ -0,0 +1,56 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
+
+  def reset!
+    @cells.clear
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(x)
+   input = cells[x.to_i - 1]
+    input
+  end
+
+  def full?
+    if cells.none? {|cell| cell == " "}
+      true
+    end
+  end
```

Comment:
Here, we can use Ruby's implicit return rather than an `if` statement.  This could be simplified to:
``` ruby
def full?
  cells.none? {|cell| cell == " "}
end
```